src/components/session.js

```diff
@@ -106,7 +116,12 @@ class WebsocketConnection extends React.PureComponent {
   render () {
     if (!this.state.connected) {
       if (this.state.connecting) {
-        return 'Connecting to web socket connection'
+        return (
+          <div className={this.props.classes.progress}>
```

Comment:
Can we please do an `const { classes } = this.props` here, since the other Components use this pattern all over the app :)